### ISSUE TITLE
Custom Event Name Variables from Booking Questions

### ISSUE THREAD

**Issue by rawkode** [calcom/cal.com#12824]
### Is your proposal related to a problem?

I’d like to be able to customise event names from the booking questions. 

Specifically, if there’s a question that asks for their company name; I’d want to use that for the booking “Discovery Call with XYZ”

### Describe the solution you'd like

(Describe your proposed solution here.)

### Describe alternatives you've considered

(Write your answer here.)

### Additional context

(Write your answer here.)

### Requirement/Document

(Share it here.)

---
##### House rules
- If this issue has a `🚨 needs approval` label, don't start coding yet. Wait until a core member approves feature request by removing this label, then you can start coding.
  - For clarity: Non-core member issues automatically get the `🚨 needs approval` label.
  - Your feature ideas are invaluable to us! However, they undergo review to ensure alignment with the product's direction.

**Comment by PeerRich** [comment]
duplicate of https://github.com/calcom/cal.com/issues/12921 will prioritise this higher
